docs/source/aryn_cloud/aryn_partitioning_service.rst

```diff
@@ -1,16 +1,18 @@
 Aryn Partitioning Service
 =========================
 
-You can use the Aryn Partitioning Service to segment PDFs into labeled bounding boxes identifying titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
+You can use the Aryn Partitioning Service to segment PDFs, Microsoft Word (.docx and .dox) documents, and Microsoft PowerPoint (.pptx and .ppt) presentations and to automatically extract table data.  The service labels titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
```

Comment:
The first sentence sounds clunky.

Reply:
How about "You can use the Aryn Partitioning Service to segment documents into labeled bounding boxes identifying titles, tables, table rows and columns, images, and regular text. The service can segment PDFs, Microsoft Word (.docx and .dox) documents, and Microsoft PowerPoint (.pptx and .ppt) file formats."

Reply:
+1

Reply:
Done.